microovn/cmd/microovn/service_control.go

```diff
@@ -47,6 +47,8 @@ func (c *cmdDisable) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	fmt.Printf("Service %s disabled\n", targetService)
+	warnings, err := client.GetServiceWarningsString(context.Background(), cli)
+	fmt.Printf(warnings)
```

Comment:
Should these warnings go to `STDERR`?

Reply:
i dont think so, they make sense in stdout